test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -14,7 +14,7 @@
   > EOF
 
   $ touch bar.ml $lib.ml
-  $ dune build @check
+  $ dune build .merlin-conf/lib-"$lib"
```

Comment:
Why don't you use the `@check` alias? It should build everything that's required for merlin.

Reply:
Targeting the `.merlin-conf` path allows to verify that the config is actually on the folder that one would expect. For example, before these changes, `@check` would pass fine, but the melange artifacts could have ended in `exe-"$target"` instead of `melange-"$target"` and it would be impossible to see it on the tests.

Reply:
Yes, but all these hidden files and directories are all an implementation detail of dune. They're all subject to change and it's best not to rely on them as much as possible.

Reply:
638646dfa165d87a6c53be79789e7f141c5083b6